topics/blockchain/index.md

```diff
@@ -0,0 +1,10 @@
+---
+created_by: Satoshi Nakamoto
+display_name: Blockchain
+released: '2008'
+short_description:  Blockchain technology provides a method to make electronic payments without needing any financial institute as banks.
+topic: blockchain
+related: cryptocurrency, cryptography
+wikipedia_url: https://en.wikipedia.org/wiki/Blockchain
+---
+A blockchain is a digitized, decentralized, public ledger of all cryptocurrency transactions. Benefits of adopting blockchain technology are distributed database, lower costs transparent and incorruptible, decentralization. It cannot be corrupted altering any unit of information on the blockchain would mean using a huge amount of computing power to override the entire network.
```

Comment:
> are distributed database

Maybe "are a distributed database".

> lower costs transparent and incorruptible, decentralization

Maybe "lower costs, transparency, and decentralization". I think you can drop "incorruptible" since you mention "cannot be corrupted" in the next sentence.

I think you need a colon here: "It cannot be corrupted: altering any unit".